src/Executables/Game/Commands/UserCommand.cs

```diff
@@ -0,0 +1,25 @@
+﻿using QuantumCore.API;
+using QuantumCore.API.Game;
+using QuantumCore.API.Game.Types;
+
+namespace QuantumCore.Game.Commands;
+
+[Command("user", "List all currently connected players to this server instance")]
+public class UserCommand : ICommandHandler
+{
+    private readonly IGameServer _server;
+
+    public UserCommand(IGameServer server)
+    {
+        _server = server;
+    }
+
+    public Task ExecuteAsync(CommandContext context)
+    {
+        var message = string.Join("\n",
+            _server.Connections.Select(x => $"Lv{x.Player!.GetPoint(EPoints.Level)} {x.Player!.Name}"));
```

Comment:
Would this cause a `NullReferenceException` when a `Connection` is still in the let's say `Hanshake` phase, and no `Player` is yet assigned?

Reply:
I assume this handler is never called when not in game.
But yes you could send this packet manually and cause an exception on the server (possible DDOS vulnerability)

Reply:
Nevermind the command handlers are only called when a player exists: https://github.com/MeikelLP/quantum-core-x/blob/d02ca2b0733bd8c648764416560749dd06c125d2/src/Executables/Game/Commands/CommandManager.cs#L144